File: videoSpeed/fileSpeed.py
#!/usr/bin/env python
#! --*-- coding:utf-8 --*--
import cv2
import sys

if __name__ == '__main__' :
    
    if len(sys.argv) > 1:
        open_file = sys.argv[1]
    else:
        open_file = 0

    video = cv2.VideoCapture(open_file);
     
    # Find OpenCV version
    (major_ver, minor_ver, subminor_ver) = (cv2.__version__).split('.')
     
    if int(major_ver)  < 3 :
        fps = video.get(cv2.cv.CV_CAP_PROP_FPS)
        print("Frames per second using video.get(cv2.cv.CV_CAP_PROP_FPS): {0}".format(fps))
    else :
        fps = video.get(cv2.CAP_PROP_FPS)
        print("Frames per second using video.get(cv2.CAP_PROP_FPS) : {0}".format(fps))
    
    video.release() 

File: CPP/videoSpeed/fileSpeed.cpp
#include "opencv2/opencv.hpp"
 
using namespace cv; 
using namespace std;
 
int main(int argc, char** argv)
{
    // Open video file
    VideoCapture video("video.mp4");
 
    double fps = video.get(CV_CAP_PROP_FPS);
 
    // For OpenCV 3, you can also use the following
    // double fps = video.get(CAP_PROP_FPS);
 
    cout << "Frames per second using video.get(CV_CAP_PROP_FPS) : " << fps << endl;
     
    video.release(); 
    return 0;
} 
